README.md

```diff
@@ -70,10 +70,10 @@ Then you can run the commands below:
 yarn
 
 # serve with hot reload at localhost:9080
-yarn dev
+yarn electron:serve
 
 # build electron application for production
-yarn build
+yarn electron:build
```

Comment:
**Remember:** Adjust this file with the correct command.
(I adding this comment to resolve it just before the merge, so we don't forget)